includes/class-sensei-lesson.php

```diff
@@ -567,7 +567,7 @@ public function quiz_update( $post_id ) {
 						continue;
 					}
 
-					$value = $this->get_submitted_setting_value( $field );
+					$value = $this->get_submitted_setting_value( $field ) ?? $field['default'];
```

Comment:
It might be better for now to replace the `??` (PHP 7 syntax), so it's not blocking the release if we don't want to bump the PHP version for `3.5.2`

Reply:
Good catch! Changed here: https://github.com/Automattic/sensei/pull/3589/commits/f8d93c74c819c6c270f3456cdd3ebe11b7a6e939